notebooks/mg-20230513-00-embedding-ideas.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>maybe rewrite the title of the notebook, and the notebook name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>